bindings/java/wasmedge-jni/lib/AstModuleContext.c

```diff
@@ -0,0 +1,72 @@
+//
```

Comment:
and for all `*.c` in JNI, I think you can format them with clang-format, following the standard as `*.cpp` code in WasmEdge.